### PR TITLE
Setup and components

### PR DESCRIPTION
**Pull request by IsmailChangezi**
## Description.
- Use CRA to create react app
- Use Routers
- Make Components
- Initialize the project.
- Fix linter errors

### CODE REVIEW

**Comment by DammyShittu** [comment]
Hi @IsmailChangezi,

While you made a great effort in this project, I cannot proceed to review your code.

## Invalid Code Review Request

### Kindly remove the description texts for the sections. For example in the Tech Stack section you have this description: "Describe the tech stack and include only the relevant sections that apply to your project.". Kindly remove it and other texts of that nature in your readme file.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/components/Categories.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+
+export default function Categories() {
+  return <div>This is check</div>;
+}
```
Comment (ichala) [inline src/components/Categories.js]: - [ ] as part of the requirements kindly add a `<Button>` with the text "Check status" in your Categories page 🙏

File: src/App.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+import BookForm from './components/BookForm';
+import Categories from './components/Categories';
+import Navigation from './components/Navigation';
+
+function App() {
+  return (
+    <>
+      <Navigation />
+      <Routes>
+        <Route path="/" element={<BookForm />} />
```
Comment (ichala) [inline src/App.js]: - [ ] as part of the requirements the root path should display the book component and not BookForm kindly update it based on the detailed comment below 🙏

File: src/components/BookForm.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+
+export default function BookForm() {
+  return (
+    <form>
+      <input type={Text} placeholder="Add a title" />
+      <input type={Text} placeholder="Add an Author" />
+      <button type="submit">Submit</button>
+    </form>
+  );
+}
```
Comment (ichala) [inline src/components/BookForm.js:11]: - [ ] Create Job Creating BookForm component 👏 , but as part of the requirement it should be called inside the Books components which is missing, anyway you should follow this flow with components : 

```JS
//Books.js the main component that should be called in the '/' root
export default function Books() {
  const books = [] //add some books in the array
  return (
<>
    {books.map(data =>(
    <Book key={data.id} data={data} />
    )}
   <BookForm /> //the form you already created should be called here
</>
);
}
```

```JS
//Book.js The component that takes book data as props and displays a book
export default function Book({data}) {
  return (
<>
 // show books data here
// add delete book button as well (doesn't have to work its just mockup for now)
</>
);
}
```

Please update your code to match the requirements 🙏 

**_🛑 Note: you don't have to do exactly what i typed , its just an example to help you understand the requirements feel free to produce your own code 🙏_**

**Review by ichala — CHANGES_REQUESTED** [review]
Hi Student
Good job so far! 
👏👏👏 
You still need to work on some issues to go to the next project, but you're almost there!

the-simpsons-homer-simpson

## To Highlight 🎉

- No Linter errors  ✔️ 
- React Installed ✔️ 


 ## Required Changes ♻️


- [ ] **_🛑  Check The Comments Below_**





## Optional suggestions

- N/A

**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by frank1738 — APPROVED** [review]
Hi @IsmailChangezi , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
✅ Correct use of Gitflow
✅ No linter error
✅ Good documentation
## Optional suggestions

N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**